lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -107,6 +122,16 @@
     key: privcertkey
     cert_chain: myverytrustedchain
 
+# Server certificate upload using key from ansible-vault
+tasks:
```

Comment:
Agreed.  And I totally agree with the comment below.  Although on the counter point many modules do load from files rather than strings - cloudformation.template comes to mind (`The local path of the cloudformation template`)